doc-src/ch-pull2dir_desc.rst

```diff
@@ -0,0 +1,59 @@
+Synopsis
+========
+
+::
+
+  $ ch-pull2dir IMAGE[:TAG] DIR
+
+Description
+===========
+
+Pulls a Docker image named :code:`IMAGE[:TAG]` from Docker Hub, flattens it
+into a Charliecloud tarball and extracts the tarball into a subdirectory of 
+:code:`DIR`.
```

Comment:
Style: Write in the imperative mood, giving instructions to the computer rather than describing what is done. For example, here you'd want:

```
Pull a Docker image named :code:`IMAGE[:TAG]` from Docker Hub, flatten it
into a Charliecloud tarball, and extract ...
```

(Note also addition of Oxford comma.)